src/main/scala/uk/gov/nationalarchives/tdr/api/http/ApiServer.scala

```diff
@@ -21,7 +21,9 @@ object ApiServer extends App {
 
   scala.sys.addShutdownHook(() -> shutdown())
 
-  Http().bindAndHandle(route, "0.0.0.0", PORT)
+  val routes = new Routes(ConfigFactory.load())
```

Comment:
These changes are because I needed to inject the config object in order to test different config values. I think it's a useful change in general, though, because we're now only loading the config once rather than every time it's called.

Reply:
Good idea